vllm/config.py

```diff
@@ -246,6 +247,10 @@ def __init__(
         model_impl: Union[str, ModelImpl] = ModelImpl.AUTO,
     ) -> None:
         self.model = model
+        if hf_config_path:
+            self.hf_config_path = hf_config_path
+        else:
+            self.hf_config_path = self.model
```

Comment:
```suggestion
        self.hf_config_path = hf_config_path
```
We can avoid redundant `if` statement here.